lib/post_action_creator.rb

```diff
@@ -342,32 +346,26 @@ def create_message_creator
       raw: body,
     }
 
-    if %i[notify_moderators spam illegal].include?(@post_action_name)
```

Comment:
So before, this branch was evaluated for %i[notify_moderators spam illegal] which means that to else only notify_user was going